webapi/src/it/scala/org/knora/webapi/it/v1/KnoraSipiIntegrationV1ITSpec.scala

```diff
@@ -745,7 +745,7 @@ class KnoraSipiIntegrationV1ITSpec
       )
 
       val response: HttpResponse = singleAwaitingRequest(letterTEIRequest)
-      assert(response.status.intValue == 500)
+      assert(response.status.intValue == 404)
```

Comment:
why does this change?

Reply:
Differences Akka `Future` vs. ZIO 'Task`?

Reply:
Differences Akka `Future` vs. ZIO `Task`?